.github/workflows/ci.yaml

```diff
@@ -30,7 +32,7 @@ jobs:
           key: gradle-${{ runner.os }}-${{ hashFiles('**/*.gradle.kts') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/versions.properties') }}
 
       - name: Run Gradle Tasks
-        run: ./gradlew build koverXmlReport
```

Comment:
FYI it's not necessary to include `bin/` here - `activate-hermit` will add `bin` to the `$PATH` for you.

Reply:
awesome!